<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="header">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Выберите строки для редактирования:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="toolTip">
        <string>Создать новый объект (Ctrl+N)</string>
       </property>
       <property name="text">
        <string>&amp;Создать</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resourses/create.png</normaloff>../resourses/create.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="toolbarLayout">
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Изменить выбранный объект (F2)</string>
           </property>
           <property name="text">
            <string>&amp;Изменить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resourses/edit.png</normaloff>../resourses/edit.png</iconset>
           </property>
           <property name="shortcut">
            <string>F2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Удалить выбранные объекты (Del)</string>
           </property>
           <property name="text">
            <string>&amp;Удалить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resourses/delete.png</normaloff>../resourses/delete.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="toolTip">
            <string>Экспортировать (Ctrl+E)</string>
           </property>
           <property name="text">
            <string>&amp;Экспорт…</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resourses/save.png</normaloff>../resourses/save.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+E</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="toolTip">
            <string>Обновить объекты (F5)</string>
           </property>
           <property name="text">
            <string>&amp;Обновить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resourses/refresh.png</normaloff>../resourses/refresh.png</iconset>
           </property>
           <property name="shortcut">
            <string>F5</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Выбрано:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectedRowsCountLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>из</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalRowsCountLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
